src/components/ApiAction/ImportActionsModal.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <b-modal
+    scrollable
+    id="import-actions"
+    title="Import API Actions"
+    size="lg"
+    @hide="resetForm"
+  >
+    <template #modal-ok>
+      <b-button
+        @click="handleImport"
+        :disabled="validNames || !Boolean(files.length)"
+        variant="primary"
+        data-cy="modal-button-import-action"
+        >Import
+      </b-button>
+    </template>
+    <b-container>
+      <b-row>
```

Comment:
Div doesn't take the entire width in a container, div doesn't have a default harmonious padding/margin, use div involves to add classes like flex for example ? 
Personaly I prefer to use rows and col because I'm really familiar with the grid system... I can change it anyway